internal/configuration.go

```diff
@@ -0,0 +1,17 @@
+package internal
+
+import "github.com/caarlos0/env/v11"
+
+type Configuration struct {
+	GithubToken string `env:"GITHUB_TOKEN,notEmpty"`
+	OpenAIToken string `env:"OPENAI_API_KEY,notEmpty"`
+}
+
+func LoadConfiguration() (Configuration, error) {
```

Comment:
Consider validating the tokens further after parsing to check if they meet specific criteria, like length, format, or presence of invalid characters. This is important to avoid potential logical errors later when these tokens are used.